src/features/MonthView/styles.css

```diff
@@ -0,0 +1,32 @@
+:root {
+  --month-cell-width: calc(14.28% - 10px);
```

Comment:
If you open this style in a week you won't be able to understand what those 14.28% mean. Eather state 100 / 7 here or move it to a constant and then use it here